std/js/Lib.hx

```diff
@@ -71,6 +71,17 @@ class Lib {
 		return untyped __js__("undefined");
 	}
 
+	/**
+		Checks a value against JavaScript `undefined` value.
+
+		Note that this is only needed in very rare cases when working with external JavaScript code.
+
+		In Haxe, `null` is used to represent the absence of a value.
+	**/
+	public static inline function isUndefined<T>(value:T):Bool {
```

Comment:
Not sure if a type param is any better than just leaving `value` as `Dynamic` (or even `Any`) here.

Reply:
`Dynamic` will do, `Any` kills monomorphs:

```haxe
class Test {
  static function main() {
    trace("Haxe is great!");
  }
  static function getName(o)
    return
      if (isUndefined(o)) 'undefined';
      else o.name;//Error here: Any has no field name

  static function isUndefined(v:Any)
    return false;
}
```